h2o-algos/src/main/java/hex/deeplearning/DeepLearningModel.java

```diff
@@ -2217,6 +2226,7 @@ static void modifyParms(DeepLearningParameters fromParms, DeepLearningParameters
         // Automatically set the distribution
         if (fromParms._distribution == DistributionFamily.AUTO) {
           // For classification, allow AUTO/bernoulli/multinomial with losses CrossEntropy/Quadratic/Huber/Absolute
+          if (nClasses == 1) toParms._distribution = DistributionFamily.gaussian;
```

Comment:
is this necessary? the else branch seems to handle this case

Reply:
its wrong, thanks